tests/test_alignment.py

```diff
@@ -0,0 +1,182 @@
+from astropy.modeling import models
+from astropy import coordinates as coord
+from astropy import units as u
+from gwcs import WCS
+from gwcs import coordinate_frames as cf
+import numpy as np
+import pytest
+from stdatamodels.jwst.datamodels import ImageModel
+from roman_datamodels import datamodels as rdm
+from roman_datamodels import maker_utils as utils
+from stcal.alignment.util import (
+    compute_fiducial,
+    compute_scale,
+    wcs_from_footprints,
+)
+
+
+def _create_wcs_object_without_distortion(
+    fiducial_world=(None, None),
+    pscale=None,
+    shape=None,
+):
+    fiducial_detector = tuple(shape.value)
+
+    # subtract 1 to account for pixel indexing starting at 0
+    shift = models.Shift(-(fiducial_detector[0] - 1)) & models.Shift(
+        -(fiducial_detector[1] - 1)
+    )
+
+    scale = models.Scale(pscale[0].to("deg")) & models.Scale(
+        pscale[1].to("deg")
+    )
```

Comment:
Here `pscale` is used as a scaling factor.

Reply:
Done.